ReSharper.FSharp/src/FSharp.Psi/src/CodeStyle/FSharpFormatSettingsKey.cs

```diff
@@ -0,0 +1,51 @@
+using JetBrains.Application.Settings;
+using JetBrains.ReSharper.Psi.CodeStyle;
+using JetBrains.ReSharper.Psi.EditorConfig;
+using JetBrains.ReSharper.Psi.Format;
+
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.CodeStyle
```

Comment:
We should probably keep the old namespace here to retain existing Fantomas/code style settings.
I'm not sure if we have statistics of how many people have changed F# code style settings.